src/index.html

```diff
@@ -11,8 +11,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute is pointing to a SCSS file. It should be pointing to a CSS file instead, as browsers cannot directly interpret SCSS files. Please ensure the correct path to the compiled CSS file is used.